code/src/components/TaskList.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { tasks } from 'reducers/tasks'
+import { Checkbox } from './styles/Checkbox';
+import { IconButton } from './styles/IconButton';
+import { TaskListWrapper, SingleTaskWrapper } from './TaskListStyles';
+// for some very strange reason, I could only import these styles if i put them in the same folder as the TaskLis-component.
+
+export const TaskList = () => {
+  // FUNCTIONALITY
+  const taskList = useSelector((store) => store.tasks.items)
+  const dispatch = useDispatch()
+  const onTaskToggle = (taskId) => {
+    dispatch(tasks.actions.toggleTask({ taskId }));
+  }
+  const onRemoveTask = (taskId) => {
+    dispatch(tasks.actions.deleteTask({ taskId }));
+  }
+
+  return (
+    <TaskListWrapper>
+      <ul>
+        {taskList.map((singleTask) => {
```

Comment:
One thing I noticed is that when you have a list of several tasks and you want to add a new one with a deadline, the calendar shows up underneath the list of tasks. Perhaps something that can be fixed with z-index?